### PR TITLE
feat(o-shop): adds private routing, refactors component structure

### PR DESCRIPTION
**Pull request by mcraepv**

### CODE REVIEW

File: frontend/src/components/Shared/PrivateRoute.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { Route, Redirect } from 'react-router';
+import routePaths from '../../constants/routePaths';
+
+export const PrivateRoute = ({ component: Component, ...rest }) => (
```
Comment (aaronpaulroberts) [inline frontend/src/components/Shared/PrivateRoute.js:5]: you are exporting this two times. One here and the `default` at the end of the file

File: frontend/src/pages/Home/Home.js [hunk 2]
```diff
@@ -1,7 +1,15 @@
 import React from 'react';
+import Header from '../../components/Header/Header';
+import Main from '../../components/Main/Main';
 
 const Home = () => {
-  return <h1>Home Works!</h1>;
+  console.log('home');
```
Comment (aaronpaulroberts) [inline frontend/src/pages/Home/Home.js]: 👀

Comment (mcraepv) [reply frontend/src/pages/Home/Home.js]: !!!!!!!!!!!!!

File: frontend/src/components/Shared/PrivateRoute.js (hunk 1 above)
Comment (mcraepv) [reply frontend/src/components/Shared/PrivateRoute.js:5]: fixed

**Review by aaronpaulroberts — APPROVED** [review]

**Pull request merged**